packages/app/src/app/components/StripeMessages/UpgradeSSEToV2.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useEffects, useAppState } from 'app/overmind';
+import { MessageStripe } from '@codesandbox/components';
+import { sandboxUrl } from '@codesandbox/common/lib/utils/url-generator';
+import { hasPermission } from '@codesandbox/common/lib/utils/permission';
+
+export const UpgradeSSEToV2Stripe = () => {
+  const state = useAppState();
+  const effects = useEffects();
+  const updateSandbox = useEffects().api.updateSandbox;
+
+  return (
+    <MessageStripe variant="primary">
+      <span>
+        This sandbox runs much faster in our new Editor. Do you want to convert
+        it to a Cloud Sandbox?
+      </span>
```

Comment:
When a user has less than "write_code" permissions, we fork the sandbox instead of upgrading. Do we want to change the copy for that scenario?

cc @zehfernandes @filipelima18

Reply:
I would say that it's a good idea. I would lowercase "Editor" and change "convert" to "fork" if it's going to be a forked sandbox. I can also write some docs that we can link to from here, which we can add in another PR.